etc/picongpu/jureca-jsc/gpus_picongpu.profile.example

```diff
@@ -47,6 +48,8 @@ module load Boost/1.82.0
 module load SciPy-bundle/2023.07
 module load matplotlib/3.7.2
 module load h5py/3.9.0
+module load numba
```

Comment:
What are we loading `numba` for?

Reply:
These are dependencies of @BrianMarre's atomic physics validation scripts. They are not actively used in the project currently, but were used in the past, and may be used again in the future.

Reply:
it's a library, but otherwise correct.